etstool.py

```diff
@@ -101,19 +101,27 @@ def edm_dependencies(runtime):
 
     Returns a set of requirement strings.
     """
-    return {
-        "flake8",
-        "flake8_ets",
-        "traitsui",
+    common_dependencies = {
         "configobj",
         "coverage",
-        "importlib_resources>=1.1.0",
-        "pytables" if runtime == "3.6" else "tables",
         "pandas",
         "pyface",
-        "enthought_sphinx_theme",
-        "sphinx",
+        "tables",
+        "traitsui",
     }
+    runtime_specific_dependencies = {
+        "3.8": {
+            # Most of these are currently unavailable on Python 3.11;
+            "enthought_sphinx_theme",
+            "flake8",
+            "flake8_ets",
+            "sphinx",
```

Comment:
Again, just for context, I see that we don't have any CI jobs that build documentation, so are we using Sphinx anywhere?

Reply:
We use Sphinx to build docs after making a release, and the current workflow for that is to do it manually via etstool.

Reply:
Agreed that we _should_ ideally have a workflow for that.

Reply:
Got it, thanks!